src/frontend/screens/Library/components/InstallModal/index.tsx

```diff
@@ -182,60 +177,35 @@ export default React.memo(function InstallModal({
 
   const channelRequiresToken = !!selectedChannel?.license_config.tokens
 
-  useEffect(() => {
-    async function validateAccessCode() {
-      if (selectedChannel?.channel_id !== undefined) {
-        const result = await window.api.checkHyperPlayAccessCode(
-          selectedChannel?.license_config.id,
-          accessCode
-        )
-
-        setAccessCodeVerified(result.valid)
-
-        if (result.valid) {
-          setErrorText('')
-          setSuccessText(
-            t(
-              'hyperplay.accesscodes.success.validation',
-              'Success! Access code is valid'
-            )
-          )
-        } else {
-          setSuccessText('')
-          setErrorText(
-            t(
-              'hyperplay.accesscodes.error.validation',
-              'Access code is invalid'
-            )
-          )
-        }
-      }
-    }
-
-    if (accessCode && channelRequiresAccessCode) validateAccessCode()
-    else {
-      setErrorText('')
-      setSuccessText('')
-      setAccessCodeVerified(false)
-    }
-  }, [selectedChannel, accessCode])
-
-  const showDownloadDialog = !isSideload && gameInfo
+  const showModDialog = gameInfo && gameInfo.type === 'mod'
```

Comment:
this should be if game is market wars for now since it is not general to all mods